SPECS/ocaml-ppx-deriving.spec

```diff
@@ -0,0 +1,118 @@
+%global debug_package %{nil}
+
+Name:           ocaml-ppx-deriving
+Version:        4.1
+Release:        1%{?dist}
+Summary:        Type-driven code generation for OCaml >=4.02
+License:        MIT
+URL:            https://github.com/whitequark/ppx_deriving
+Source0:        https://github.com/whitequark/ppx_deriving/archive/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  ocaml
+BuildRequires:  ocaml-findlib
+BuildRequires:  cppo
+BuildRequires:  ocaml-ounit
+BuildRequires:  ocaml-result
```

Comment:
With the upcoming changes in `ocaml-result` you will need to use `ocaml-result-devel` here